src/ipsis/course-merge/CourseMergeOfferingBaseCollectionEntity.js

```diff
@@ -0,0 +1,23 @@
+import { BaseCollectionEntity } from './BaseCollectionEntity.js';
+
+export class CourseMergeOfferingBaseCollectionEntity extends BaseCollectionEntity {
+
+	courseMergeOfferings() {
+		if (!this._entity) {
+			return;
+		}
+		return this._entity.entities;
+	}
+
+	loadMorePageSize() {
```

Comment:
This method looks duplicated in `CourseMergeLogDetailCollectionEntity.js`. The only difference is the getter for the list length. Can you put this in `BaseCollectionEntity.js` and parameterize the getter?

Reply:
Done!